pkg/datapath/loader/netlink.go

```diff
@@ -141,3 +142,15 @@ func (l *Loader) DeleteDatapath(ctx context.Context, ifName, direction string) e
 
 	return nil
 }
+
+// HasDatapath return true if the interface as a tc filter attached on the given direction.
+func (l *Loader) HasDatapath(ctx context.Context, ifName, direction string) (bool, error) {
+	args := []string{"filter", "show", "dev", ifName, direction}
+	cmd := exec.CommandContext(ctx, "tc", args...).WithFilters(libbpfFixupMsg)
+	output, err := cmd.Output(log, true)
+	if err != nil {
+		return true, fmt.Errorf("Failed to get tc filter: %s", err)
```

Comment:
Use lowercase for the error string (see https://github.com/golang/go/wiki/CodeReviewComments#error-strings) and wrap the underlying error with `%w`:

```suggestion
		return true, fmt.Errorf("failed to get tc filter: %w", err)
```

I see other error strings in this file already use uppercase. These should probably be converted at some point as well.